code/scripting/api/objs/team.cpp

```diff
@@ -41,26 +42,27 @@ ADE_VIRTVAR(Name, l_Team, "string", "Team name", "string", "Team name, or empty
 
 ADE_FUNC(getColor,
 	l_Team,
-	nullptr,
-	"Gets the IFF color of the specified Team",
+	"boolean ReturnType",
+	"Gets the IFF color of the specified Team. True to return raw rgb, false to return color object. Defaults to true.",
 	"number, number, number",
 	"rgb color for the specified team or nil if invalid")
 {
 	int idx;
 	int r,g,b;
-	if(!ade_get_args(L, "o", l_Team.Get(&idx)))
+	bool rc = true;
+	if(!ade_get_args(L, "o|b", l_Team.Get(&idx), &rc))
 		return ADE_RETURN_NIL;
 
 	if(idx < 0 || idx >= (int)Iff_info.size())
 		return ADE_RETURN_NIL;
 
-	color* col = iff_get_color_by_team(idx, 0, 0);
-
-	r = col->red;
-	g = col->green;
-	b = col->blue;
+	color* cur = iff_get_color_by_team(idx, 0, 0);
 
-	return ade_set_args(L, "iii", r, g, b);
+	if (rc) {
+		return ade_set_args(L, "iii", (int)cur->red, (int)cur->green, (int)cur->blue, (int)cur->alpha);
```

Comment:
As with `getIFFColor`, the number of characters does not match the number of variables, but this could also officially be bumped to four.

Reply:
Changed